### PR TITLE
Remove obsolete Polyfill packages and configuration

### PR DESCRIPTION
**Pull request by IonBazan**
Removes unused `symfony/polyfill*` packages and `incenteev-parameters` configuration.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -127,6 +127,11 @@
...
         "symfony/polyfill-ctype": "*",
         "symfony/polyfill-iconv": "*",
+        "symfony/polyfill-php81": "*",
+        "symfony/polyfill-php80": "*",
+        "symfony/polyfill-php74": "*",
+        "symfony/polyfill-php73": "*",
+        "symfony/polyfill-php72": "*",
         "symfony/polyfill-php71": "*",
         "symfony/polyfill-php70": "*",
         "symfony/polyfill-php56": "*"
```
Comment (localheinz) [inline composer.json:137]: What about these?

Comment (IonBazan) [reply composer.json:137]: These are in `replace` section, so they will not be installed even if other packages require them.

Comment (localheinz) [reply composer.json:137]: Ha, my bad!

👍

**Review by localheinz — APPROVED** [review]
👍

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**